bin/crew

```diff
@@ -1964,15 +1909,7 @@ def install_command(args)
 end
 
 def list_command(args)
-  if args['available']
-    list_available
-  elsif args['installed']
-    puts list_installed
-  elsif args['compatible']
-    list_compatible true
-  elsif args['incompatible']
-    list_compatible false
-  end
+  Command.list(args['available'], args['installed'], args['compatible'], args['incompatible'], @opt_verbose)
```

Comment:
Shouldn't need to pass `@opt_verbose` since it is a global instance variable.

Reply:
Global variables and instance variables are different-- `@opt_verbose` is accessible to everything in `bin/crew`, but we are crossing a class barrier and thus instance variables do not propagate.